### PR TITLE
add trayplay 1.0.3

### PR DESCRIPTION
**Pull request by timbroder**

### CODE REVIEW

File: Casks/trayplay.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'trayplay' do
+  version '1.0.3'
+  sha256 '00da4099a040fa50e9f73f13ab217309e8c9a3683834164a401f638db0246331'
+
+  url 'http://mborgerson.com/trayplay/dist/TrayPlay-1.0.3.dmg'
```
Comment (victorpopkov) [inline Casks/trayplay.rb]: You can use string interpolation here to reuse the version number. An [appcast](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md) also is available and can be specified just after the `url` with appropriate `checkpoint`:
```ruby
  url "http://mborgerson.com/trayplay/dist/TrayPlay-#{version}.dmg"
  appcast 'https://mborgerson.com/trayplay/dist/appcast.xml',
          checkpoint: '0a9f111bef597bfa7f800a1f7d016049d9f59bb207d0e0d1c04e34d35d18dd66'
```

Comment (timbroder) [inline Casks/trayplay.rb]: Thanks for pointing that out @victorpopkov updating now

**Comment by victorpopkov** [comment]
Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by victorpopkov** [comment]
Thank you for the contribution. It was merged directly as 4b78f9548d26319f97be7f4cdf29d30d5a07696b to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**